core/src/main/resources/jenkins/install/pluginSetupWizard.properties

```diff
@@ -56,3 +58,20 @@ installWizard_pluginInstallFailure_message=Some plugins failed to install proper
 with the failed plugins
 installWizard_continue=Continue
 installWizard_retry=Retry
+installWizard_upgradePanel_title=Upgrade
+installWizard_upgradePanel_banner=Welcome to Jenkins {0}!
+installWizard_upgradePanel_message=Jenkins {0} includes some great new features that we think you'll love, install these additional plugins to take advantage of them!
+installWizard_upgradePanel_recommendedActionTitle=Install Suggested Plugins
+installWizard_upgradePanel_recommendedActionDetails=Install the suggested plugins
```

Comment:
The redundancy here is somewhat goofy but I don't have a better suggestion at this time.